### PR TITLE
Assert Less Than for Status Check

### LINKED ISSUE #1315: Assert Less Than for Status Check
Similar to #1314 this check will perform `<` instead of `<=` with key  ``assert_lt`` that follows same pattern.

### PR DESCRIPTION
**Pull request by Xiangs18**
Hi @shahzebsiddiqui, please review this MR and provide feedback.

pic

### CODE REVIEW

**Comment by shahzebsiddiqui** [comment]
@prathmesh4321 can you please review

File: buildtest/buildsystem/checks.py [hunk 1]
```diff
@@ -478,6 +478,77 @@ def assert_gt_check(builder):
     return bool_check
 
 
+def assert_lt_check(builder):
+    """Perform check on assert greater than when ``assert_lt`` is specified in buildspec. The return is a boolean value that determines if the check has passed.
```
Comment (prathmesh4321) [inline buildtest/buildsystem/checks.py]: Should be "Perform check on assert less than....."

File: buildtest/buildsystem/checks.py [hunk 2]
```diff
@@ -478,6 +478,77 @@ def assert_gt_check(builder):
...
+        bool_check = conv_value < ref_value
+        console.print(
+            f"[blue]{builder}[/]: testing metric: {name} if {conv_value} < {ref_value} - Check: {bool_check}"
+        )
+        assert_check.append(bool_check)
+
+    # perform a logical AND on the list and return the boolean result
+    bool_check = all(assert_check)
+
+    console.print(f"[blue]{builder}[/]: Greater Check: {bool_check}")
```
Comment (prathmesh4321) [inline buildtest/buildsystem/checks.py]: Should be Less than check.

**Review by prathmesh4321 — CHANGES_REQUESTED** [review]
@Xiangs18 Can you also create a symlink in https://github.com/buildtesters/buildtest/tree/devel/tests/builders for the assert less than test in https://github.com/buildtesters/buildtest/tree/devel/tutorials/perf_checks if not done yet, so that the example test also gets tested. See https://github.com/buildtesters/buildtest/pull/1361 for reference.

**Comment by prathmesh4321** [comment]
> @prathmesh4321 can you please review

@shahzebsiddiqui Sure!

**Comment by Xiangs18** [comment]
Hey @prathmesh4321, updated!

**Comment by shahzebsiddiqui** [comment]
thanks @Xiangs18 this looks good. thanks @prathmesh4321 for reviewing this. I resolved all threads. Merging this in 🚀

**Pull request merged**